indico/modules/events/papers/templates/display/_paper_list.html

```diff
@@ -15,38 +15,66 @@
                                     {% if reviewed or to_review -%}
-                                        <div class="submitter icon-user" data-searchable="{{ latest_revision.submitter.display_full_name|lower }}">
-                                            {{ latest_revision.submitter.display_full_name }}
+                                        <div class="submitter icon-user align-icon"
+                                             data-searchable="{{ latest_revision.submitter.display_full_name|lower }}">
+                                            {{- latest_revision.submitter.display_full_name -}}
                                         </div>
-                                        <div>
-                                            {% if session.user in contribution.paper_layout_reviewers %}
-                                                <div class="label i-tag layout-reviewer review-role">
-                                                    {%- trans %}Layout{% endtrans -%}
-                                                    {% if latest_revision.has_user_reviewed(session.user, 'layout') %}
-                                                        <i class="icon-circle-small" title="{% trans %}Already reviewed{% endtrans %}"></i>
+                                        {% set state_icon_map = {'submitted': 'icon-file-eye',
+                                                                 'accepted': 'icon-checkmark',
+                                                                 'rejected': 'icon-cross',
+                                                                 'to_be_corrected': 'icon-tilde'} %}
+                                        <div class="{{ state_icon_map[latest_revision.state.name] }} align-icon semantic-text {{ latest_revision.state.css_class }}">
+                                            {{- latest_revision.state.title -}}
+                                            {% if latest_revision.state.name == 'submitted' %}
+                                                {% set review_type %}
+                                                    {% if session.user in contribution.paper_layout_reviewers and session.user in contribution.paper_content_reviewers %}
+                                                        {% trans %}for content and layout review{% endtrans %}
+                                                    {% elif session.user in contribution.paper_content_reviewers %}
+                                                        {% trans %}for content review{% endtrans %}
+                                                    {% elif session.user in contribution.paper_layout_reviewers %}
+                                                        {% trans %}for layout review{% endtrans %}
                                                     {% endif %}
+                                                {% endset %}
+                                                {{ review_type }}
```

Comment:
You don't really need a set block here

```suggestion
                                                {% if session.user in contribution.paper_layout_reviewers and session.user in contribution.paper_content_reviewers %}
                                                    {% trans %}for content and layout review{% endtrans %}
                                                {% elif session.user in contribution.paper_content_reviewers %}
                                                    {% trans %}for content review{% endtrans %}
                                                {% elif session.user in contribution.paper_layout_reviewers %}
                                                    {% trans %}for layout review{% endtrans %}
                                                {% endif %}
```

Reply:
However, we are now concatenating two translatable strings (state title + this).

Better move the "Submitted" into the trans strings here, and move showing just the state title into the `else` of `{% if latest_revision.state.name == 'submitted' %}`

Reply:
Yes, I was suspecting splitting that string was not a good idea. :+1: